CMakeLists.txt

```diff
@@ -94,6 +96,11 @@ endif()
 add_subdirectory(libraries)
 add_subdirectory(examples)
 
+if (MORPHEUS_BUILD_WITH_CCACHE)
+    set(MORPHEUS_LANGUAGES CXX)
```

Comment:
Actually, the are also parts of Objective C/C++ in the Metal redendering library and I'm also planning for Cuda support in the future.

Perhaps we should move this out of the if statement and define this as a top-level variable for the project.

Reply:
Perfect I will move it out of the if statement. The reason why I created a dedicated variable `MORPHEUS_LANGUAGES` is that unfortunately the project command option `LANGUAGES` [doesn't set any variable](https://cmake.org/cmake/help/v3.25/command/project.html):
```
project(<PROJECT-NAME>
        [VERSION <major>[.<minor>[.<patch>[.<tweak>]]]]
        [DESCRIPTION <project-description-string>]
        [HOMEPAGE_URL <url-string>]
        [LANGUAGES <language-name>...])
```
I will move it out of the if statement and rename it as `CCACHE_LANGUAGES` and pass it to the function.